docs/app/Examples/elements/Input/Variations/Size.js

```diff
@@ -0,0 +1,34 @@
+import React, {Component} from 'react';
+import {Input} from 'stardust';
+
+export default class Size extends Component {
+  render() {
+    return (
+      <div>
+        <Input className='mini icon' placeholder='Search...'>
+          <i className='search icon' />
+        </Input>
+        <br />
+        <Input className='small icon' placeholder='Search...'>
+          <i className='search icon' />
+        </Input>
+        <br />
+        <Input className='large icon' placeholder='Search...'>
+          <i className='search icon' />
+        </Input>
+        <br />
+        <Input className='big icon' placeholder='Search...'>
+          <i className='search icon' />
+        </Input>
+        <br />
+        <Input className='huge icon' placeholder='Search...'>
+          <i className='search icon' />
+        </Input>
+        <br />
+        <Input className='massive icon' placeholder='Search...'>
+          <i className='search icon' />
+        </Input>
+      </div>
+    );
+  }
+}
```

Comment:
Wasn't sure the best way to handle this example. I put all the size examples in one file with `br` between them. Alternate ideas would be create separate files.. eg.. `SizeMini.js`, `SizeBig.js` and then put them all in the Size example section. Thought it was 50/50 on adding lots of files vs putting all this in one. Open to suggestions